package.json

```diff
@@ -38,33 +44,54 @@
     "cli"
   ],
   "dependencies": {
-    "JSONStream": "^1.3.2",
-    "agentkeepalive": "^3.3.0",
-    "algoliasearch": "^3.24.9",
+    "JSONStream": "^1.3.5",
+    "agentkeepalive": "^4.0.0",
+    "algoliasearch": "^3.31.0",
     "async": "^2.6.0",
     "batch-stream": "^0.1.3",
     "chalk": "^2.4.1",
-    "commander": "^2.18.0",
+    "commander": "^2.19.0",
     "csvtojson": "^2.0.8",
-    "event-stream": "3.3.4",
-    "json-text-sequence": "^0.1.1",
-    "stream-json": "^0.5.2",
-    "stream-transform": "^1.0.0"
+    "event-stream": "4.0.1",
+    "stream-transform": "^1.0.7"
   },
   "devDependencies": {
-    "babel-eslint": "^9.0.0",
+    "babel-eslint": "^10.0.1",
     "babel-jest": "^23.6.0",
-    "dotenv": "^6.1.0",
-    "eslint": "^5.6.0",
+    "dotenv": "^6.2.0",
+    "eslint": "^5.9.0",
     "eslint-config-algolia": "^13.2.3",
-    "eslint-config-prettier": "^3.0.1",
+    "eslint-config-prettier": "^3.3.0",
     "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jest": "^21.22.0",
-    "eslint-plugin-prettier": "^2.6.2",
+    "eslint-plugin-jest": "^22.1.2",
+    "eslint-plugin-prettier": "^3.0.0",
     "jest": "^23.6.0",
-    "prettier": "^1.14.2"
+    "prettier": "^1.15.3",
+    "rimraf": "^2.6.2",
+    "striptags": "^3.1.1"
+  },
+  "eslintConfig": {
+    "extends": [
+      "algolia",
+      "algolia/jest"
+    ],
+    "rules": {
+      "import/no-commonjs": "off",
+      "import/extensions": "ignorePackages",
+      "no-console": "off",
+      "no-process-exit": "off",
+      "prettier/prettier": [
+        "error",
+        {
+          "trailingComma": "es5",
```

Comment:
why not in a .prettierrc so it works without eslint too?

Reply:
I never settled on the exact right way to have prettier in a project + eslint actually so dunno here!